src/Entity/Tradeability/Shipment.php

```diff
@@ -83,11 +83,6 @@ public function toNode(DOMDocument $document = null)
 
         $node = $document->createElement('Shipment');
 
-        // Objects first
-        if ($this->getAdditionalInsurance() instanceof AdditionalInsurance) {
```

Comment:
Well the getter/setter could stay there, it's just never an instance of `AdditionalInsurance`.